sirepo_bluesky/shadow_handler.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+import Shadow.ShadowTools
+
+
+def read_shadow_file(filename, parameter=11):
+    """Read specified parameter from the Shadow3 output binary file.
+
+    Parameters
+    ----------
+    filename : str
+        Shadow3 output binary file (.dat)
+    parameter : int
+        The parameter to read (11=Energy [eV] by default)
```

Comment:
Consider using parameter 23 (https://github.com/oasys-kit/shadow3/blob/d007ae59a2038db4f9275f7bb026bd1b11549e5f/Shadow/ShadowTools.py#L68) for the total intensity.